src/pages/send/confirm/confirm.ts

```diff
@@ -294,31 +294,30 @@ export class ConfirmPage extends WalletTabsChild {
     }
   }
 
-  private setWalletSelector(
-    coin: string,
-    network: string,
-    minAmount: number
-  ): Promise<any> {
+  private setWalletSelector(coin: string, network: string): Promise<any> {
     const parentWallet = this.getParentWallet();
     if (
       parentWallet &&
       (parentWallet.network == network && parentWallet.coin == coin)
     ) {
       return Promise.resolve();
     }
-    return new Promise(resolve => {
-      // no min amount? (sendMax) => look for no empty wallets
-      minAmount = minAmount ? minAmount : 1;
-
-      this.wallets = this.profileProvider.getWallets({
-        onlyComplete: true,
-        hasFunds: true,
-        network,
-        coin
-      });
 
-      return resolve();
+    this.wallets = this.profileProvider.getWallets({
+      onlyComplete: true,
+      hasFunds: true,
+      network,
+      coin
```

Comment:
shouldn't we be adding `minAmount` here as one of the opts, so the wallet selector will not show any non eligible wallets?

Reply:
```
this.wallets = this.profileProvider.getWallets({
      onlyComplete: true,
      hasFunds: true,
      network,
      coin,
      minAmount <----
```

Reply:
mmm personally I don't like that solution, I want to see all my wallets and check balances of each of them. Not showing the wallets might cause confusion on users that doesn't know why they can't see their wallets in case the first wallet have enough funds but the second no ( the second one will not be shown with no explanation )